### ISSUE TITLE
Count CCN for reminder_handler@156-190@./packages/reminder.py

### ISSUE THREAD

**Issue by remnestal** [remnestal/Jarvis#1]
As part of the assignment, the CCN has to be counted by hand for reminder_handler@156-190@./packages/reminder.py (output from _coverage.py_)

**Comment by remnestal** [comment]
My calculations yielded:
- **23 nodes**
- **30 edges**
- **1 component**

Meaning that the CCN is **8** or **9**, depending on if you use the _M = E − N + P_ or the _M = E − N + 2P_ formula. Neither value is the same as the CCN value of **10** that [lizard ](https://pypi.python.org/pypi/lizard) yielded.

Please see [the flowchart I made for this calculation.](https://drive.google.com/open?id=1G6gikInXarFKyxiYzo0oyCIn2nwTPx_e)

**Comment by joberg92** [comment]
My flowchart: https://drive.google.com/open?id=15NwW3-pIB1HcjonHk0iHYLjR-lBqISW2 

- 20 Nodes
- 27 Edges
- 1 Component
=> M= E - N + 2*P = 27-20+2*1= **9** or 8 in the case of just P and not 2*P

**Comment by remnestal** [comment]
Great, you're using _"implicit"_ for-loop index incrementation right? That would explain why our number of nodes and edges differ by 3 each. I have a hard time seeing why we get a different answer than _lizard_, do you have any clue @joberg92?

**Comment by joberg92** [comment]
Yes I used implicit next incrementation for the for each loop. Im not sure why lizard gets 1 more. Maybe lizard is using another formula for CCN? Good thing we get same results at least!
